code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -37,12 +37,12 @@
 /mob/living/simple_animal/hostile/syndicate/Die()
 	..()
 	if(corpse)
-		new corpse (src.loc)
+		corpse.createCorpse()

```

Comment:
My code just compiled but yes it should, the only problem that can occur will come from the loc thats used.